### ISSUE TITLE
working with 32bit or 16 bit floating point images for ACES color managed workflow.

### ISSUE THREAD

**Issue by luuude** [wanmeihuali/taichi_3d_gaussian_splatting#145]
As far as I can understand this implementation uses only 8bit images as input. Would it be possible to take linear EXR or similar formats to preserve more data from the camera sensor?

**Comment by wanmeihuali** [comment]
Indeed, you can definitely use linear EXR or similar high-bit-depth formats as input. While the current implementation predominantly handles colors normalized into 32-bit floats, modifying the input transformation handled by PyTorch should be straightforward. However, you might need to tweak certain parameters, particularly during rasterization. For instance, values like 1./255., set based on pixel sensitivity, might require lowering to accommodate the higher fidelity data.

**Comment by luuude** [comment]
That is great news! Being a beginner in pytorch and python could you point in the direction of where to look to modiify the input transformation?
